.github/workflows/ci.yml

```diff
@@ -174,17 +156,14 @@ jobs:
         run:
           echo "::set-output name=version::$(rustc -V)"
 
-      # Current size as of 2019-12-23: ~336 MB
+      # Current size as of 2021-02-12: ~325 MB
       - name: Cache cargo build
-        uses: actions/cache@v2
+        uses: actions/cache@main
         with:
           path: target
-          key: ${{ runner.os }}-cargo-build-target-${{ steps.rustc.outputs.version }}-${{ hashFiles('**/Cargo.lock') }}
+          key: v2-${{ runner.os }}-cargo-build-target-${{ steps.rustc.outputs.version }}-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
do we still need this cache bump?

Reply:
I'm not sure if we need to or not. I think the bins from `cargo install` are the only cache key that hasn't changed, and while we probably don't have to change this key it will help ensure that we're evaluating the performance from a clean slate.

Reply:
okay, sounds reasonable 👍